### PR TITLE
fix(input-date-picker): ensure range input toggling is consistent

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #6501 

## Summary

This fixes an issue where the start/end picker would not update after being toggled and displayed on the toggled side.

### CODE REVIEW

File: packages/calcite-components/src/components/input-date-picker/resources.ts [hunk 1]
```diff
@@ -1,6 +1,15 @@
 export const CSS = {
   assistiveText: "assistive-text",
+  calendarWrapper: "calendar-wrapper",
+  calendarWrapperEnd: "calendar-picker-wrapper--end",
```
Comment (eriklharper) [inline packages/calcite-components/src/components/input-date-picker/resources.ts:4]: ☝️ should the value here omit the word `picker` to be more inline with the key pattern?

Comment (jcfranco) [reply packages/calcite-components/src/components/input-date-picker/resources.ts:4]: Great catch! 🎣 Fixing.

File: packages/calcite-components/src/components/input-date-picker/input-date-picker.tsx [hunk 2]
```diff
@@ -532,15 +534,15 @@ export class InputDatePicker
                 aria-describedby={this.placeholderTextId}
                 aria-expanded={toAriaBoolean(this.open)}
                 aria-haspopup="dialog"
-                class={`input ${
-                  this.layout === "vertical" && this.range ? `no-bottom-border` : ``
-                }`}
+                class={{
+                  [CSS.input]: true,
+                  [CSS.inputNoBottomBorder]: this.layout === "vertical" && this.range,
+                }}
```
Comment (eriklharper) [inline packages/calcite-components/src/components/input-date-picker/input-date-picker.tsx:540]: 🧹

**Review by eriklharper — APPROVED** [review]
LGTM! 👍

**Pull request merged**